Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/MarkerDetection/ArUco/HoloLens/SpectatorViewPluginArUcoMarkerDetector.cs

```diff
@@ -360,8 +360,8 @@ private sealed class StationaryMarkerDetectionCompletionStrategy : MarkerDetecti
             private const int _requiredObservations = 5;
             private const int _requiredInlierCount = 5;
             private const int _maximumMarkerSampleCount = 15;
-            private const float _maximumPositionDistanceStandardDeviation = 0.001f;
-            private const float _maximumRotationAngleStandardDeviation = 0.25f;
+            private const float _maximumPositionDistanceStandardDeviation = 0.01f;
+            private const float _maximumRotationAngleStandardDeviation = 0.75f;
```

Comment:
It seems like it may be worth exposing these values as serialized fields. Users may always need to manipulate these values and it will become more difficult when binaries are distributed via nuget.